frontend/src/tests/utils.js

```diff
@@ -36,6 +36,23 @@ expect.extend({
     }
 });
 
+/**
+ * Customize the `.toMatchObject` method to print the contents of the recieved
+ * object if the response call fails
+ *
+ * @export
+ * @param {object} resp The recieved array
+ * @returns
+ */
+export function toMatchSuccessDebug(resp) {
+    try {
+        expect(resp).toMatchObject({ status: "success" });
+    } catch (err) {
+        console.warn("resp: \n", resp);
+        throw err;
```

Comment:
This is still not right. The error message needs to be descriptive, and you shouldn't use console.warn.

Reply:
Ok think I finally got it. I was missing `JSON.stringify`

Reply:
This is improved! But I think you can do better. Imagine you, in January, trying to work on your first bug. What error message would you hope to see? Write that message.

Also, the description of this function is not good. In particular, it doesn't explain why anyone should use this function.